src/Identity/Extensions.Core/src/Base32.cs

```diff
@@ -46,7 +46,7 @@ public static byte[] FromBase32(string input)
             input = input.TrimEnd('=').ToUpperInvariant();
             if (input.Length == 0)
             {
-                return new byte[0];
+                return Array.Empty<byte>();
```

Comment:
Why do we even have this class?!

Reply:
I hear you have to use base32 if you are on a 32-bit processor.

Reply:
This was mostly just to get the finicky authenticator apps to work properly with TFA, its possible we don't need this helper anymore, but that's why it was added originally